### ISSUE TITLE
An assembly error

### ISSUE THREAD

**Issue by zhangrengang** [chhylp123/hifiasm#110]
![graph-1](https://user-images.githubusercontent.com/22489982/117282927-dadad000-ae97-11eb-936e-6f208842b4e3.png)
This is a real example in unitig graph (`*.hic.p_utg.gfa`) that seems to be an error: the overlap between A and B is only 408 bp, but the paths A-B and A-C-B are resolved in `*.hic.hap1.p_ctg.gfa` and `*.hic.hap2.p_ctg.gfa`, respectively. Nevertheless, it is ok in `*.hic.p_ctg.gfa` (A-C-B). I used default parameters with HiC mode.

**Comment by chhylp123** [comment]
Thanks a lot. I guess the reason is that hifiasm thinks it is a bubble but it is actually not...

**Comment by chhylp123** [comment]
Are A, B and C homozygous unitigs or not?

**Comment by zhangrengang** [comment]
No, they are not homozygous, as there are some heterozygous small bubbles in A, B and C (see the graph). It seems that `-x` and `-y` did not work. I think the short overlap should be re-checked by reads that span it.

**Comment by chhylp123** [comment]
If A and B are at two haplotypes, does it mean hifiasm generate false duplication for A and B?
